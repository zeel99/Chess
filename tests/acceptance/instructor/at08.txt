setup_chess(N, 1, 2)

start_game

-- only one chess piece to start with: the user wins immediately!

-- Error: game already over
moves(1, 2)
