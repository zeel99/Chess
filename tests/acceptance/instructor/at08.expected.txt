  # of chess pieces on board: 0
  Game being Setup...
  ....
  ....
  ....
  ....
->setup_chess(N,1,2)
  # of chess pieces on board: 1
  Game being Setup...
  .N..
  ....
  ....
  ....
->start_game
  # of chess pieces on board: 1
  Game Over: You Win!
  .N..
  ....
  ....
  ....
->moves(1,2)
  # of chess pieces on board: 1
  Error: Game already over
  .N..
  ....
  ....
  ....
